src/helpers/namespace.ts

```diff
@@ -46,5 +46,124 @@ export type CreateManyInput<T> =
       set?: T | T[];
     };
 
+/**
```

Comment:
This file is getting a little big, would you mind to move to a .ts file and import it as string using `fs.readFile`? This way typescript also checks it.

Reply:
Sure, great call

Reply:
@arthurfiorette so I tried to implement this on a different branch: https://github.com/replohq/prisma-json-types-generator/compare/noahsark769/issue-34-string-enum-unions...replohq:prisma-json-types-generator:noahsark769/try-reading-from-file?expand=1 - it turns out it's a bit more complicated than I imagined because the namespace generation happens on `prisma generate`, which means we're in the `dist` folder and no longer have access to the original ts source, and trying to read from the generated js source doesn't work because the file is completely type definitions, so the generated js doesn't have any content.

I don't have a ton of experience developing npm packages, but as far as I can tell the original ts source is not included when installing. Maybe we could add some sort of build-step to copy in the .ts file but that seems quite a lot more complicated. Maybe makes sense to do in a followup, unless there's some easier way here that I'm missing? I was able to get it to typecheck correctly and found two instances of the T -> S replacement that I missed, though!

Reply:
You can create this file inside SRC, read it from `../src/path` import (assume `./` is in dist folder), and maybe use something like `$$NAMESPACE` as a placeholder so that you can `.replace()` it later.

TIP: With `npm publish --dry`, you can see what files are uploaded or not.

Reply:
It makes even more sense to store the generated into a `template` folder.

Reply:
Got more free time now, I'll do it 4 u

Reply:
nvm, i don't know if it was a bug with pnpm or so, but it just stopped working at all. YAGNI is the best principle to follow rn :)